src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,13 +8,24 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String EXAMPLE = "123, some street, #01-01, 123456";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person's address must be entered in the format a/BLOCK, STREET, UNIT, POSTAL_CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
+    public static final int NUM_ARGS_ADDRESS = 4;
+    public static final int INDEX_BLOCK = 0;
+    public static final int INDEX_STREET = 1;
+    public static final int INDEX_UNIT = 2;
+    public static final int INDEX_POSTAL_CODE = 3;
```

Comment:
I don't think it should be declared as `static` variable. For the index, you can check `enum`. Good job for removing magic numbers.